components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -116,10 +122,18 @@ func NewReconciler(ctx context.Context, natsSubMgr subscriptionmanager.Manager,
 		Client:     client,
 		logger:     logger,
 		record:     recorder,
-		cfg:        cfg,
+		cfg:        backendCfg,
 	}
 }
 
+func (r *Reconciler) SetNatsConfig(natsConfig env.NATSConfig) {
```

Comment:
Why is this public? Where is this used?

Reply:
It is used in Eventing Auth Manager. Eventing Manager updates NATS config to apply the desired NATS configuration from Eventing CR.

Reply:
Thank you!